recipes/treecorr/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "TreeCorr" %}
+{% set version = "3.3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e0acf0283fa3f5677061358621f6ae0fd7cc6bda8bfced1343256dd4f127b366
+
+build:
+  number: 0
+  skip: True  # [win]
+  script:
+    # the setup.py file does some fancy testing for being able to compile
+    # things right - these env vars are needed for toolchain builds to make
+    # that work
+    - export CPATH="${PREFIX}/include"  # [linux]
+    - export LIBRARY_PATH="${PREFIX}/lib"  # [linux]
```

Comment:
Shouldn't these be prepends, not replacements.

```
export CPATH="${PREFIX}/include:${CPATH}"
```

Reply:
As far as I know the toolchain packages on Linux are not setting these vars so I made them replacements. This is easy enough to change though.

Reply:
It is a good idea to not do replacements like this just in case. I thought CPATH was touched, but in the event that it isn't, think of it as future proofing :wink: